### PR TITLE
Update plan_capacity.md

### PR DESCRIPTION
**Pull request by mdelder**
Closes open-cluster-management/backlog#2342.

+ Clarify sizing requirements. 
+ More details available in https://docs.google.com/presentation/d/1YzLMLvUXju5DW-IU2HgIhkJ7oLQhXCCzbXdsNX5ATeY/edit#slide=id.g88133efd1a_0_13

### CODE REVIEW

**Review by dockerymick — APPROVED** [review]
Apologies for the misunderstanding, thanks for the changes Michael! I made a couple of changes, otherwise the PR looks good to me :)

**Comment by mdelder** [comment]
Thanks @dockerymick !

File: install/plan_capacity.md [hunk 1]
```diff
@@ -2,54 +2,62 @@
...
+  
 
+  - Node Count: 3
+  - Availability Zones: 3
   - Instance size: N1-standard-4 (0.95–6.5 GB)
-  - vCPU: 6
-  - Memory: 16 GB
-  - Storage size: 120 GB  
+    - vCPU: 4
+    - Memory: 15 GB
```
Comment (cdoan1) [inline install/plan_capacity.md:44]: Hi Michael, is this a typo. Not worried about 1.0, more for how this is carried over to 2.0. Looking for a clearly defined statement that minimal nodes are 4x16.

Comment (mdelder) [reply install/plan_capacity.md:44]: The original document listed instance sizes for each cloud but then listed invalid capacities for them; for instance, the N1-standard-4 size only has 15Gi vs. 16Gi. 

Our actual memory reservation in 2.0 is 9.3Gi, but what we really care more about is that the nodes are distributed across 3 AZs. So we should be more than fine with 3 nodes of 15Gi in 3 AZs.

**Comment by mdelder** [comment]
@dockerymick When will this change make it into 2.0_stage/prod?

**Pull request merged**